README.md

```diff
@@ -84,87 +94,92 @@ Inside database.yml add username and password for development and test:
 
-### Google Places API (optional)
+Some tips for your first interactions with the Teams App. For this, make sure you have the server running and the app open in your browser 🖥.
 
-To avoid accidentally exceeding the rate limit on [Google's Places API][google-places] (e.g. when heavily testing city-autocomplete fields) and thus blocking its usage for other RGSoC sites and devs:
+    http://localhost:3000
 
-1. [Get your own key][google-places]
+To access all functionality of the app, add yourself as an organizer. For this, first start an interactive Rails console in a separate terminal (window/tab) 📟:
 
-1. Copy the `.env-example` file to `.env` and replace `YourGoogleMapsAPIKey` with your key
+    bundle exec rails console
 
-1. Run the `foreman` command before any command
-  ```sh
-  # e.g.
-  foreman run rails server
-  # or
-  foremand run rails console
+1. In the browser 🖥: log in with your Github account
+1. In the console 📟
+
+  ```ruby
+  user = User.last
+  user.roles.create(name: 'organizer')
+
+  # alternative: find your user, e.g. by your Github handle
+  user = User.find_by(github_handle: <your-github-handle-here>)
+
+  # You can assign yourself other roles the same way.
+  # If you've assigned yourself student AND organizer roles however,
+  # this may lead to undesired behavior of the app. Best remove the
+  # student role again.
   ```
-[google-places]: https://developers.google.com/places/javascript/
-
-## Quick Start 
-
-### Beginner Friendly Tips for New Contributors
-
-- After forking the repo, follow the steps described above under 'Bootstrap'. Mailtrap is optional.
-- (Install and) connect to Postgres server 
-- With everything properly installed, open the browser in development environment
-- The app should be available, with the database loaded with fake data.
-- To access all the functionality of the teams app, add yourself as an organizer.
-    * In the browser: log in with your github account 
-    * In Rails Console:
-      ```
-      user = User.last
-      user.roles.create(name: "organizer")
-      ```
-      Instead of using the last created User, you can also search for a
-      specific User with ```user = User.find_by(github_handle: "yourgithubhandle")```.
-
-      You can assign yourself other roles in the same way as creating the
-      organizer role. If however you assign yourself a student role AND another
-      role, that may lead to unexpected behavior in the app. In that case,
-      remove the student role. 
-    * Refresh the browser and you should see links for organizers. 
-- Once you are an `organizer`, you can add a season and switch between season's phases at
-http://localhost:3000/organizers/seasons in your browser.
-- While in development, you are also able to impersonate other users to easily test the system
-as someone else. Go to http://localhost:3000/users while logged in to do that.
-- You are good to go now. Happy coding!
+1. Refresh the browser 🖥: you should see links to orga pages now
```

Comment:
Interestingly, Github is rendering this `1.` as `1.` and not as `3.` as I would expect it at least. Maybe it's because of the code block above?